File: tests/cgi/cgi-test.py
#!/usr/bin/python3

import os

print ("Content-Type: text/html\r")
print ("\r")
print ("<!DOCTYPE html>")
print ("<html>")
print ("<head>")
print ("<title>decoded text</title>")
print ("</head>")
print ("<body><h1>")
print ("caesar(plain_text)")
print ("<table>")
for key, val in os.environ.items():
    print("<tr><td>", key, "</td><td>", val, "</td></tr>")
print ("</table>")
print ("</h1></body>")
print ("</html>")

File: tests/cgi/cgi-test.swift
#!/usr/bin/swift

import Foundation

print("Content-Type: text/html\n\n")
print("<html><body>")
print("<h1>CGI Test</h1>")
print("<p>Environment Variables:</p>")
print("<table>")
for (key, value) in ProcessInfo.processInfo.environment {
    print("<tr><td>\(key)</td><td>\(value)</td></tr>")
}
print("</table>")
print("<p>Arguments:</p>")
print("<table>")
for arg in CommandLine.arguments {
    print("<tr><td>\(arg)</td></tr>")
}
print("</table>")
print("</body></html>")

